.github/workflows/release.yml

```diff
@@ -72,3 +69,13 @@ jobs:
           asset_path: ${{ format('chart.js-{0}.tgz', needs.setup.outputs.version) }}
           asset_name: ${{ format('chart.js-{0}.tgz', needs.setup.outputs.version) }}
           asset_content_type: application/gzip
+      - name: Deploy To latest tag
```

Comment:
Does this run on top of the previous output? If so, will this work? Two CI steps would change the `docs/.vuepress/config.js` file but the 2nd `sed` in this step would fail since `VERSION` would be already replaced.